theme/theme.scss

```diff
@@ -0,0 +1,17 @@
+// Copyright (c) 2003-2017, CKSource - Frederico Knabben. All rights reserved.
+// For licensing, see LICENSE.md or http://ckeditor.com/license
+
+@import '~@ckeditor/ckeditor5-theme-lark/theme/theme.scss';
+
+// TODO move to make a common style with editor-classic.
```

Comment:
I created an [issue](https://github.com/ckeditor/ckeditor5-theme-lark/issues/97) for this. We'll fix it as a follow-up.